backend/src/test/java/team/teamby/teambyteam/member/docs/MemberApiDocsTest.java

```diff
@@ -0,0 +1,361 @@
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.delete;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.post;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.preprocessRequest;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.preprocessResponse;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.prettyPrint;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseBody;
+import static org.springframework.restdocs.request.RequestDocumentation.parameterWithName;
+import static org.springframework.restdocs.request.RequestDocumentation.pathParameters;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(MemberController.class)
+public final class MemberApiDocsTest extends ApiDocsTest {
+
+    @MockBean
+    private MemberService memberService;
+
+    @Nested
+    @DisplayName("내 정보 조회 조회 문서화")
+    class RetrieveMyInformation {
+
+        @Test
+        @DisplayName("내 정보 조회 성공")
+        void success() throws Exception {
+            // given
+            final Long id = 1L;
+            final String name = "name";
+            final String profileUrl = "/";
+            final String email = "email@email.com";
+            given(memberService.getMemberInformation(any()))
+                    .willReturn(new MemberInfoResponse(id, name, profileUrl, email));
+
+            // when & then
+            mockMvc.perform(get("/api/me")
+                            .header(AUTHORIZATION_HEADER_KEY, AUTHORIZATION_HEADER_VALUE)
+                            .contentType(MediaType.APPLICATION_JSON))
+                    .andExpect(status().isOk())
+                    .andDo(print())
+                    .andDo(document("members/retrieveMyInformation/success",
+                                    preprocessRequest(prettyPrint()),
+                                    preprocessResponse(prettyPrint()),
+                                    requestHeaders(
+                                            headerWithName(AUTHORIZATION_HEADER_KEY).description("사용자 JWT 인증 정보")
+                                    ),
+                                    responseBody()
+                            )
+                    );
+        }
+
+        @Test
+        @DisplayName("인증되지 않은 사용자면 실패한다.")
+        void failIfUnAuthorized() throws Exception {
+            // given
+            given(memberInterceptor.preHandle(any(), any(), any()))
+                    .willThrow(AuthenticationException.FailAuthenticationException.class);
```

Comment:
확인해보니 이런식으로 하면 문서화된 adoc파일을 보니 예외메시지가 null로 생성이 되네요.

```suggestion
given(memberInterceptor.preHandle(any(), any(), any()))
        .willTrhow(new AuthenticationException.FailAuthenticationException());
```

이렇게 변경해주시면 문서에서도 어떤 식으로 예외가 발생되는지 알 수 있을 것 같습니다.